doc/components.md

```diff
@@ -248,7 +252,7 @@ A more in-depth example can be found [here](/doc/examples/isometric).
 
 ![An example of a isometric map with selector](images/isometric.png)
 
-# Nine Tile Box Component
+## NineTileBoxComponent
```

Comment:
would be cool to have a diagram with the inheritance tree!

Reply:
Very good idea!

Reply:
Added